src/components/TogetherButton/TogetherButton.stories.js

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import TogetherButton from './TogetherButton';
+
+storiesOf('TogetherButton', module).add('default', () => <TogetherButton />);
```

Comment:
Can you explain a little bit what a story is used for?

Reply:
That's a good point, I wasn't paying close attention to this when I set it up. And I just googled it seems that it's about some project-independent testing. I think we can get rid of this. I'll recommit

Reply:
It will probably be some learning curve so I don't think we need this for now.